tracer/src/Datadog.Trace.ClrProfiler.Native/rejit_handler.cpp

```diff
@@ -298,6 +302,42 @@ void RejitHandler::RequestRejit(std::vector<ModuleID>& modulesVector,
     }
 }
 
+void RejitHandler::RequestRevert(std::vector<ModuleID>& modulesVector, std::vector<mdMethodDef>& modulesMethodDef)
+{
+    if (IsShutdownRequested())
+    {
+        return;
+    }
+
+    HRESULT hr;
+
+    if (!modulesVector.empty())
+    {
+        // *************************************
+        // Request Revert
+        // *************************************
+
+        if (m_profilerInfo10 != nullptr)
+        {
+            HRESULT *status = nullptr;
+            hr = m_profilerInfo10->RequestRevert((ULONG) modulesVector.size(), &modulesVector[0], &modulesMethodDef[0], status);
+        }
+        else
+        {
+            HRESULT* status = nullptr;
+            hr = m_profilerInfo->RequestRevert((ULONG) modulesVector.size(), &modulesVector[0], &modulesMethodDef[0], status);
+        }
```

Comment:
Hmm, that's what I've noticed we were doing with `RequestReJIT` so I followed the same pattern.
https://github.com/DataDog/dd-trace-dotnet/blob/0e23585745d965ea143036238a6033ed931c09bd/tracer/src/Datadog.Trace.ClrProfiler.Native/rejit_handler.cpp#L288-L293

Do you think I should simplify it to `m_profilerInfo->RequestRevert`?

Reply:
`RequestReJIT` follow that pattern because the intention is to have a different call, in this case the plan is to enable `RequestReJITWithInliners` when version 10 is being used. I don’t know if this is going to be the case for Revert…

Reply:
Yep. I see that now. It was pretty confusing. I'll simplify that.

Reply:
Fixed in 7359e7819949e3530d3fa1c6076beab0fa9b3422.

Reply:
Great!